yarn.lock

```diff
@@ -4214,6 +4214,43 @@
     prop-types "^15.6.1"
     react-lifecycles-compat "^3.0.4"
 
+"@react-stately/grid@3.0.0-alpha.2":
```

Comment:
hey! something has gone wrong with your package.json's these shouldn't be in the yarn.lock
double check that the version you have as a dependency in your package.json matches the local workspace package